source/core-developers/accessing-application-session-request-objects.md

```diff
@@ -50,16 +50,24 @@ is an alternative way to access the request and response objects, with the same
 ## Accessing from the view (JSP, FreeMarker, etc.)
 
 Request and session attributes are accessed via OGNL using the `#session` and `#request` stack values.
+Page attributes are accessed via OGNL using the `#attr` stack value, and Application attributes via
+the `#application` stack value.
 
 The `#attr` stack value will search the `javax.servlet.jsp.PageContext` for the specified key. If the `PageContext`
-doean't exist, it will search the request, session, and application scopes, in that order.
+doesn't exist, it will search the request, session, and application scopes, in that order.
 
-**Accessing the Session or Request from a JSP**
+**Accessing attributes in the Application, Session, Request, or Page scope from a JSP**
 
 ```jsp
+Retrieve the attribute (property), with key myId, from the specified scope:
```

Comment:
Nice to use xml comment here i.e. `<!-- .... -->`

Reply:
This is a JSP page, not XML. I would rather wrap this line (and the on below) with `<p></p>`